ui/components/EventMembers/index.tsx

```diff
@@ -114,11 +118,30 @@ const EventMembers = ({ collection, currentUser }) => {
       fetching: loading,
       error,
     },
+    searchApprovedMembers
   ] = useQuery({
     query: COLLECTION_MEMBERS_QUERY,
-    variables: { collectionId: collection.id, offset: 0, limit: 1000 },
+    variables: { collectionId: collection.id, usernameStartsWith: searchString, offset: 0, limit: 1000 },
```

Comment:
@aerugo is it enough for now that we search by username or do we want e.g. name and/or email as well?

Reply:
Lets do username for now, then add name and email later.